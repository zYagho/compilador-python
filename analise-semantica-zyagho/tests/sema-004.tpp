{Aviso: Variável 'b' declarada e não utilizada}
{Aviso: Variável 'a' declarada e inicializada, mas não utilizada}
{Erro: Função 'principal' deveria retornar 'inteiro', mas retorna 'vazio'}

inteiro: a 
inteiro: b

inteiro principal()
	a := 10
fim
