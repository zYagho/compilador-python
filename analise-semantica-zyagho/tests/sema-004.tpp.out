WAR-SEM-VAR-DECL-NOT-USED
WAR-SEM-VAR-DECL-INIT-NOT-USED
ERR-SEM-FUNC-RET-TYPE-ERROR
